extension/data_loader/file.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
```

Comment:
Please add a comment describing the purpose of this header: i.e., it ensures that a `pread`-compatible function is defined in the global namespace for windows and posix environments.

Reply:
Updated.